front/src/components/AsideNav/AsideNav.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classes from './AsideNav.module.css';
+import {Link} from "react-router-dom";
+import {Box, List, ListItemIcon} from "@mui/material";
+import ListItem from "@mui/material/ListItem";
+import SupportAgentIcon from "@mui/icons-material/SupportAgent";
+import ChatIcon from "@mui/icons-material/Chat";
+import ListItemText from "@mui/material/ListItemText";
+import HomeIcon from "@mui/icons-material/Home";
+import InfoIcon from "@mui/icons-material/Info";
+import navigationStore from "../../stores/navigationStore";
+import clsx from "clsx";
+import {observer} from "mobx-react-lite";
+
+const AsideNav = observer(() => {
+
+  const className = clsx(classes.root, {[classes.open]: navigationStore.open});
+
+  return (
+    <>
+      <div className={classes.static}></div>
+      <Box sx={{color: "#505050", fontWeight: "bold", boxShadow: 1}} className={className}>
```

Comment:
Since we are creating a separate css file, can these parameters be specified in them?